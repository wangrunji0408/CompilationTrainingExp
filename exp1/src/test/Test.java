package test;

public class Test {
    public static void main (String [] args) {
	int a, b, c;
	a = 3;
	if (a < 2) {
	    b = 3;
	    c = 5;
	} else {
	    b = 3;
	    c = 6;
	}
	a = -a;
	c = a+b;
	System.out.println(a+" "+b+" "+c);
    }
}
