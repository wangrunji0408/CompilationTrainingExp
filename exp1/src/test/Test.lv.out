Now analyzing test.Test
Method: main
Initialization completed.
entry: []
1 in:  []
1 out: []
2 in:  []
2 out: []
3 in:  []
3 out: []
4 in:  []
4 out: []
5 in:  []
5 out: []
6 in:  []
6 out: []
7 in:  []
7 out: []
8 in:  []
8 out: [T4]
9 in:  [T4]
9 out: [R1]
10 in:  [R1]
10 out: [R1, T4]
11 in:  [R1, T4]
11 out: [R1, R3]
12 in:  [R1, R3]
12 out: [R1, R3, T5]
13 in:  [R1, R3, T5]
13 out: [R1, R3, T5, T6]
14 in:  [R1, R3, T5, T6]
14 out: [R1, R3, T5, T6, T7]
15 in:  [R1, R3, T5, T6, T7]
15 out: [R1, R3, T5, T6]
16 in:  [R1, R3, T5, T6, T7]
16 out: [R1, R3, T5, T6, T7]
17 in:  [R1, R3, T5, T6]
17 out: [R3, T5, T6]
18 in:  [R1, R3, T5, T6]
18 out: [R1, R3, T5, T6]
19 in:  [R3, T5, T6, T7]
19 out: [R3, T5, T6]
20 in:  [R3, T5, T6]
20 out: [R3, T5, T6, T7]
21 in:  [R3, T5, T6, T7]
21 out: [R3, T5, T6, T7]
22 in:  [R3, T5, T6, T8]
22 out: [R3, T5, T6]
23 in:  [R3, T5, T6]
23 out: [R3, T5, T6, T8]
24 in:  [R3, T5, T6, T8]
24 out: [R3, T5, T6, T8]
25 in:  [R3, T5, T6, T7]
25 out: [R3, T5, T6]
26 in:  [R3, T5, T6]
26 out: [R3, T5, T6, T7]
27 in:  [R3, T5, T6, T7]
27 out: [R3, T5, T6, T7]
28 in:  [R3, T5, T6]
28 out: [T5, T6]
29 in:  [R3, T5, T6]
29 out: [R3, T5, T6]
30 in:  [T5, T6]
30 out: [T5, T6]
31 in:  [T5, T6]
31 out: [T5, T6]
32 in:  [T5, T6]
32 out: []
33 in:  [T5, T6]
33 out: [T5, T6]
34 in:  []
34 out: []
exit: []
Method: <init>
Initialization completed.
entry: [R0]
1 in:  [R0]
1 out: []
2 in:  [R0]
2 out: [R0]
3 in:  []
3 out: []
exit: []
